script/upgrades/eXOF/Config.sol

```diff
@@ -0,0 +1,203 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// solhint-disable func-name-mixedcase, contract-name-camelcase, function-max-lines, var-name-mixedcase
+pragma solidity ^0.5.13;
+pragma experimental ABIEncoderV2;
+
+import { Chain } from "script/utils/Chain.sol";
+import { Config } from "script/utils/Config.sol";
+import { Contracts } from "script/utils/Contracts.sol";
+import { Arrays } from "script/utils/Arrays.sol";
+import { FixidityLib } from "script/utils/FixidityLib.sol";
+
+/**
+ * @dev This library contains the configuration required for the eXOF governance proposal.
+ *      The following configuration is used:
+ *     - 2 pools: eXOFCelo and eXOFEUROC
+ *     - 2 rate feeds: CELOXOF and EURXOF
+ *     - Configuration params needed to initialize the eXOF stable token
+ */
+library eXOFConfig {
+  using FixidityLib for FixidityLib.Fraction;
+  using Contracts for Contracts.Cache;
+
+  struct eXOF {
+    // Pools
+    Config.Pool eXOFCelo;
+    Config.Pool eXOFEUROC;
+    Config.Pool[] pools;
+    // Rate Feeds
+    Config.RateFeed CELOXOF;
+    Config.RateFeed EURXOF;
+    Config.RateFeed[] rateFeeds;
+    Config.StableToken stableTokenXOF;
+  }
+
+  /**
+   * @dev Returns the populated configuration object for the eXOF governance proposal.
+   */
+  function get(Contracts.Cache storage contracts) internal returns (eXOF memory config) {
+    config.pools = new Config.Pool[](2);
+    config.pools[0] = config.eXOFCelo = eXOFCelo_PoolConfig(contracts);
+    config.pools[1] = config.eXOFEUROC = eXOFEUROC_PoolConfig(contracts);
+
+    config.rateFeeds = new Config.RateFeed[](2);
+    config.rateFeeds[0] = config.CELOXOF = CELOXOF_RateFeedConfig(contracts);
+    config.rateFeeds[1] = config.EURXOF = EURXOF_RateFeedConfig(contracts);
+
+    config.stableTokenXOF = stableTokenXOFConfig();
+  }
+
+  /* ==================== Rate Feed Configurations ==================== */
+
+  /**
+   * @dev Returns the configuration for the CELOXOF rate feed.
+   */
+  function CELOXOF_RateFeedConfig(
+    Contracts.Cache storage contracts
+  ) internal view returns (Config.RateFeed memory rateFeedConfig) {
+    rateFeedConfig.rateFeedID = contracts.deployed("StableTokenXOFProxy");
+    rateFeedConfig.medianDeltaBreaker0 = Config.MedianDeltaBreaker({
+      enabled: true,
+      threshold: FixidityLib.newFixedFraction(3, 100), // 0.03
+      cooldown: 30 minutes,
+      smoothingFactor: 0
+    });
```

Comment:
```suggestion
     });
    rateFeedConfig.dependentRateFeeds = Arrays.addresses(contracts.dependency("EUROXOFRateFeedAddr"));
```

Reply:
did this change in to also add the deployment checks https://github.com/mento-protocol/mento-deployment/pull/115/commits/43d6531699733a363529496b401a0d2f34d6ade9

Reply:
mistakenly used `EUROCEUR` as the dependency while it should be `EURXOF`, fixed here https://github.com/mento-protocol/mento-deployment/pull/115/commits/da68e08b850b90e27ad0303cf29543391605ff2b